x-pack/packages/ml/aiops_utils/window_parameters.ts

```diff
@@ -15,25 +15,13 @@ import { isPopulatedObject } from '@kbn/ml-is-populated-object';
  * @typedef {WindowParameters}
  */
 export interface WindowParameters {
-  /**
```

Comment:
Removed the JSDoc type annotations here because it's overly verbose and isn't needed to satisfy the docs script. There's also not really an IDE/Intellisense benefit when the file is TS anyway.